build/Dockerfile.ci

```diff
@@ -0,0 +1,101 @@
+# CI Dockerfile
+# CI requires a special Dockerfile for use with OpenShift Builds which is what is used to build the image. This requires
+# building the operator from the PR source without using the operator-sdk.
+# NOTE: RUN mkdir <dir> is being used in cases where a directory needs to be created as CI image builds does not seem to
+# be creating the directory as a result of the WORKDIR directive. 
+# build stage for building binaries
+FROM registry.access.redhat.com/ubi8/go-toolset:1.12.8-32 as build
+LABEL stage=build
+
+# Build WMCB
+# WORKDIR is not resulting in creating this directory, so we are forcing it to happen. The image does not allow us to
```

Comment:
Please add a TODO or comment saying that this will also be a download in future when we cut releases.

Reply:
I don't plan to ever move to downloading WMCB but rather pin to a branch like `release-4.5`. I can add a comment that says that.